Assignment2/r090047.rdf

```diff
@@ -0,0 +1,10 @@
+@base <http://www.oeg.upm.net/ontologies/classresources>
```

Comment:
Those qNames starting with : will generate strange URIs. It is good practice to use / or # at the endo of the base prefix (and other prefixes)